tests/Container/MySqlProjectionManagerFactoryTest.php

```diff
@@ -0,0 +1,50 @@
+<?php
+/**
+ * This file is part of the prooph/pdo-event-store.
+ * (c) 2016-2017 prooph software GmbH <contact@prooph.de>
+ * (c) 2016-2017 Sascha-Oliver Prolic <saschaprolic@googlemail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ProophTest\EventStore\Pdo\Container;
+
+use PHPUnit\Framework\TestCase;
+use Prooph\EventStore\EventStore;
+use Prooph\EventStore\Pdo\Container\MySqlProjectionManagerFactory;
+use Prooph\EventStore\Pdo\Projection\MySqlProjectionManager;
+use ProophTest\EventStore\Pdo\TestUtil;
+use Psr\Container\ContainerInterface;
+
+/**
+ * @group pdo_pgsql
+ */
+final class MySqlProjectionManagerFactoryTest extends TestCase
```

Comment:
exclude this class in coverage, see https://github.com/prooph/pdo-event-store/blob/master/phpunit.xml.pdo_pgsql#L20-L27